python/semantic_kernel/connectors/memory/mongodb_atlas/mongodb_atlas_memory_store.py

```diff
@@ -266,7 +266,7 @@ async def get_nearest_matches(
         pipeline.append({"$set": {"score": {"$meta": "vectorSearchScore"}}})
 
         if min_relevance_score:
```

Comment:
Can score be negative? If so, this should be `min_relevance_score is not None` to handle the zero case?